config/jobs/kubernetes-sigs/sig-windows/windows-unit-presubmits.yaml

```diff
@@ -35,3 +35,42 @@ presubmits:
       testgrid-dashboards: sig-windows-presubmit
       testgrid-tab-name: pull-windows-unit-master
       testgrid-num-columns-recent: '20'
+
+  - name: pull-ci-kubernetes-unit-windows-presubmit
+    always_run: true
+    optional: true
+    decorate: true
+    decoration_config:
+      timeout: 3h
```

Comment:
this is excessive for a presubmit, if it takes this long, it should not be in presubmit.

Reply:
A full run (the one above) runs for about 100 minutes. This job runs only a subset, but I'm not sure how long it takes. We could reduce this timeout to 2h for now, and we are thinking of ways to shorten the duration. From what I saw, it takes a while to run the unit tests for the first time, but subsequent ``go test`` are shorter.

Reply:
100 minutes is still way too long for a presubmit. We consider above 30m to be a regression and are working on those, above 1h is pretty much unacceptable ...

Is it a matter of machine size? Or is there some reason this is fundamentally slow?